stubs/2.7/time.pyi

```diff
@@ -0,0 +1,48 @@
+"""Stub file for the 'time' module."""
+# based on autogenerated stub from typeshed and https://docs.python.org/2/library/time.html
+
+from typing import NamedTuple, Tuple, Union
+
+# ----- variables and constants -----
+accept2dyear = False 
+altzone = 0
+daylight = 0
+timezone = 0
+tzname = ... # type: Tuple[str, str]
+
+struct_time = NamedTuple('struct_time',
+                         [('tm_year', int), ('tm_mon', int), ('tm_mday', int),
+                          ('tm_hour', int), ('tm_min', int), ('tm_sec', int),
+                          ('tm_wday', int), ('tm_yday', int), ('tm_isdst', int)])
+
+_TIME_TUPLE = Tuple[int, int, int, int, int, int, int, int, int]
+
+def asctime(t: Union[_TIME_TUPLE, struct_time, None] = None) -> str:
```

Comment:
A few minor issues: `_TIME_TUPLE` is not needed since `NamedTuple[...`] is a subtype of the corresponding tuple type. No need to use `Union[..., None]` for a default argument value if the default value `None`. The `Union` is inferred automatically.